.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,29 +14,40 @@ Changelog:
 ## Types of changes
 
 What types of changes does your code introduce to Zowe Explorer?
-_Put an `x` in the boxes that apply_
+_Put an `x` in all boxes that apply._
 
-- [ ] Bugfix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Enhancement (non-breaking change which adds or improves functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
The words "fix or feature" might be able to be removed to make this line shorter.